docs/release/trg-4/trg-4-1.md

```diff
@@ -2,43 +2,43 @@
 title: TRG 4.01 - Image tagging
 ---
 
-| Author               | Status | Created     | Post-History                     |
-|----------------------|--------|-------------|----------------------------------|
-| Catena-X System Team |        | 10-Nov-2022 | Initial release                  |
-| Catena-X System Team | Active | 24-Nov-2022 | more precise process description |
+| Status | Created     | Post-History                     |
+|--------|-------------|----------------------------------|
+| Active | 24-Nov-2022 | more precise process description |
+|        | 10-Nov-2022 | Initial release                  |
 
-## Description
+## Why
 
-The build process for Docker images must ensure proper image tagging. All images must be tagged
-following [semantic versioning](https://semver.org/), e.g. `:0.1.2` as well as the latest image must be tagged in
-addition with `:latest`.
+Using non-specific image tags like `:latest` would require everyone to find and check long hashes instead of a version number.
 
-## Why
+This also break control of selecting specific versions and upgrading as `:latest` would pull in new images on restart of a pod.
 
-We do that to easily identify the application version running in your container.
+Container image tagging also helps you to match [TRG 5.05 - Chart Values](../trg-5/trg-5-5.md#container-images).
 
-As these container images will be used in versioned Helm charts, using `:latest` as tag would destroy the idea of a
-versioned and released Helm chart, as a changed image could be injected without changing the Helm chart version. Proper
-Docker image tagging also helps you to match [TRG 5.05](../trg-5/trg-5-5.md#container-images).
+## Description
+
+The build process for container images **must** ensure proper image tagging. All images **must** be tagged
+following [semantic versioning](https://semver.org/), e.g. `:0.1.2` as well as the latest image must be tagged in
+addition with `:latest`.
 
-### Process description
+### Process example description
 
-If you follow our recommendation how to create Docker images and tag them, the following list describes the process
+If you follow our recommendation how to create container images and tag them, the following list describes the process
 steps:
 
 1. Create/edit your `Dockerfile` and merge Changes via PullRequest to `main` branch, if changes where made in a feature
    branch, or similar.
-2. Create/push a Git tag to `main` branch, e.g.:
+1. Create/push a Git tag to `main` branch, e.g.:
```

Comment:
```suggestion
2. Create/push a Git tag to `main` branch, e.g.:
```

Reply:
In markdown its getting auto numbered when using 1. 1. 1.. Nummering manually is not a good idea

Reply:
But these are steps following each other. Maybe then switch to unordered list ( - ) instead of ordered.